### ISSUE TITLE
Titlecasing sometimes not helpful.

### ISSUE THREAD

**Issue by operations-smco** [marekrei/smuploader#5]
I found 
https://github.com/marekrei/smuploader/blob/6b1726fdb49529952e103b3de9428fd92fdecf0c/smuploader/smugmug.py#L206

was a bit problematic in that I didn't want titlecasing, so dropped .title() from the code.

Would be good to have a user driven option to turn this on or off.

**Comment by marekrei** [comment]
True. However, because the "nice name" has many restrictions, the original string needs to be modified anyway, and it's subjective what are the exact best modifications.

I'll leave this comment open as a suggestion for future work.
